lib/db.js

```diff
@@ -512,19 +514,30 @@ module.exports = (
       lastAccessTime: token.lastAccessTime,
       createdAt: token.createdAt,
       isMemoryToken: true
-    }]
+    }
     let sessionTokens = []
+    return getGeoData(ip)
```

Comment:
Did we measure this? What were the results? Is it worth adding a note about perf to the readme for [mozilla/fxa-geodb](https://github.com/mozilla/fxa-geodb) in case we wonder about it again in the future?